File: SI.java
import java.util.Scanner;

public class SimpleInterest {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter Principal Amount: ");
        float principal = sc.nextFloat();

        System.out.print("Enter Rate of Interest: ");
        float rate = sc.nextFloat();

        System.out.print("Enter Time (in years): ");
        float time = sc.nextFloat();

        float SI = (principal * rate * time) / 100;

        System.out.println("Simple Interest = " + SI);

        sc.close();
    }
}

File: SI.c
#include <stdio.h>

int main() {
    float principal, rate, time, SI;

    printf("Enter Principal Amount: ");
    scanf("%f", &principal);

    printf("Enter Rate of Interest: ");
    scanf("%f", &rate);

    printf("Enter Time (in years): ");
    scanf("%f", &time);

    SI = (principal * rate * time) / 100;

    printf("Simple Interest = %.2f", SI);

    return 0;
}
